scripts/yapf_formatter.py

```diff
@@ -0,0 +1,31 @@
+from yapf.yapflib.yapf_api import FormatCode
+from github import Github
+import sys
+import re
+import base64
+
+ACCESS_TOKEN = []
+REPO_NAME = []
+WORKING_PATH = []
+ACCESS_TOKEN = sys.argv[1]
+REPO_NAME = sys.argv[2]
+WORKING_PATH = sys.argv[3]
+
+g = Github(ACCESS_TOKEN)
+repo = g.get_repo(REPO_NAME)
+contents = repo.get_contents(WORKING_PATH)
+
```

Comment:
If you have time, better refactor this python script.

Example can be found [here](https://github.com/BobAnkh/add-contributors/blob/master/main.py).

Also, you can choose to refactor all the python scripts in this repo in another new pr.